bedrock/firefox/tests/test_base.py

```diff
@@ -1121,22 +1121,17 @@ def setUp(self):
                       'Gecko/20100101 Firefox/35.0')
         self.url = reverse('firefox.hello.start', args=['35.0'])
 
-    def test_fx_hello_no_conversation(self):
-        """Should identify when there is no conversation"""
+    def test_fx_hello_redirect_non_firefox(self):
+        """Should redirect to /firefox/new if not on Firefox"""
```

Comment:
Because `HelloStartView` uses the `LatestFxView` base class, I believe we already have tests for this [here](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/tests/test_base.py#L954). Although maybe @pmclanahan can better confirm than i can